config/detekt.yml

```diff
@@ -562,6 +562,7 @@ style:
     active: false
   MaxLineLength:
     active: true
+    excludes: ['**/Activity.kt']
```

Comment:
Why is this excluded? Were there problems with the detekt rules?

Reply:
detekt meckert wahrscheinlich an den zu langen Zeilen, was jedoch nicht so viel Sinn macht, weil das ein autogeneriertes File aus dem devtools ist.

Reply:
Ok, Danke